templates/modern-solutions/rop-shell/statement.rst

```diff
@@ -0,0 +1,12 @@
+Open the session task archive and access the ``rop-shell/`` subfolder. Your
+goal is to open a shell by calling ``system("/bin/sh")``. You can jump
+directly to where ``system()`` is called. You need to store the address of
+``sh`` in RDI using a gadget. You can find the address of ``sh`` in the
+executable by using find in GDB PEDA.
```

Comment:
Use \`\`find\`\` for formatting as it's a GDB PEDA keyword.

Reply:
Will do.